lib/drivers/common/prepare.js

```diff
@@ -47,13 +47,19 @@ function spawn(dir, env, cmd, callback) {
   var options = {
     cwd: dir,
     env: _.extend({}, process.env, env),
-    stdio: 'inherit',
+    stdio: debug.enabled ? ['ignore', 'pipe', 'pipe'] : 'ignore',
```

Comment:
I was thinking something like:
```
stdio: ['ignore', debug.enabled ? 'pipe' : 'ignore', 'pipe'],
```

when prepare fails, the app isn't deployed... but its undebuggable, the user has no idea what went wrong, so I think stderr from npm should always be in our logs

stdout is boring and noisy, and we never care